proto/autokitteh/remote/v1/remote.proto

```diff
@@ -0,0 +1,229 @@
+// This is temporarly here until we find a good home for it
```

Comment:
ticket. also this file needs some big big big comment explaining what is going on here. what talks to what? when? why?

some kind of ascii diagram maybe.

Reply:
the diagram is part of the workflow isolation design documents
everything is described there in the mermaid diagram

do we need to add it here as well  ?